orangecontrib/network/widgets/OWNxHist.py

```diff
@@ -208,11 +208,11 @@ def generateGraph(self, N_changed=False):
             graph.add_nodes_from(range(self.matrix.shape[0]))
             matrix = self.matrix
 
-            if hasattr(self.matrix, "items") and self.matrix.items is not None:
-                if isinstance(self.matrix.items, Orange.data.Table):
-                    graph.set_items(self.matrix.items)
+            if hasattr(self.matrix, "row_items") and self.matrix.row_items is not None:
```

Comment:
Dropping hasattr has no effect; the error still passes silently.

In order to avoid such problems in the future, I recommend more careful testing before committing.